win32ss/user/user32_vista/dpi.c

```diff
@@ -42,3 +43,81 @@ GetDpiForWindow(
     UNREFERENCED_PARAMETER(hWnd);
     return GetDpiForSystem();
 }
+
+/*
+ * @stub
+ */
+BOOL
+WINAPI
+IsProcessDPIAware(void)
+{
+    UNIMPLEMENTED;
+    return FALSE;
+}
+
+/*
+ * @stub
+ */
+BOOL
+WINAPI
+SetProcessDPIAware(void)
+{
+    UNIMPLEMENTED;
+    return FALSE;
+}
+
+/*
+ * @stub
+ */
+BOOL
+WINAPI
+SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT context)
```

Comment:
If the prototypes of these functions don't come from Wine, I would suggest adding SAL2 annotations. Otherwise they can stay as they are.